### PR TITLE
Add new version in ga.tests.ui.pr

### PR DESCRIPTION
**Pull request by tleon**
| Questions         | Answers
| ------------------| -------------------------------------------------------
| Description?      | Add new version in ga tests ui 
| Type?             |new feature 
| BC breaks?        | no
| Deprecations?     | no
| Sponsor company   | PrestaShop SA
| How to test?      | CI green

### CODE REVIEW

File: .github/workflows/actions/checkout-prestashop/action.yml [hunk 1]
```diff
@@ -58,6 +58,6 @@ runs:
 
     # Workaround until https://github.com/PrestaShop/PrestaShop/issues/29813 is fixed
     - name: PrestaShop Configuration (Copy of Config API)
-      if: (inputs.base_branch == '8.1.x' || inputs.base_branch == '8.2.x')
+      if: (inputs.base_branch == '8.1.x' || inputs.base_branch == '8.2.x' || inputs.base_branch == '9.0.x' )
```
Comment (Progi1984) [inline .github/workflows/actions/checkout-prestashop/action.yml:61]:
```suggestion
      if: (inputs.base_branch == '8.1.x' || inputs.base_branch == '8.2.x')
```

File: .github/workflows/test-sanity.yml [hunk 2]
```diff
@@ -201,7 +201,7 @@ jobs:
       # Playwright must be installed all the time, we only install chromium since it's the only browser used
       - name: Install browsers
         working-directory: ${{ env.PS_DIR }}/tests/UI
-        if: ${{ inputs.base_branch == '8.0.x' || inputs.base_branch == '8.1.x' || inputs.base_branch == '8.2.x' }}
+        if: ${{ inputs.base_branch == '8.0.x' || inputs.base_branch == '8.1.x' || inputs.base_branch == '8.2.x' || inputs.base_branch == '9.0.x' }}
```
Comment (Progi1984) [inline .github/workflows/test-sanity.yml:204]:
```suggestion
        if: ${{ inputs.base_branch == '8.0.x' || inputs.base_branch == '8.1.x' || inputs.base_branch == '8.2.x' }}
```

File: .github/workflows/test-with-prebuilt-shop.yml [hunk 3]
```diff
@@ -171,7 +171,7 @@ jobs:
       # Playwright must be installed all the time, we only install chromium since it's the only browser used
       - name: Install browsers
         working-directory: ${{ env.PS_DIR }}/tests/UI
-        if: ${{ inputs.base_branch == '8.0.x' || inputs.base_branch == '8.1.x' || inputs.base_branch == '8.2.x' }}
+        if: ${{ inputs.base_branch == '8.0.x' || inputs.base_branch == '8.1.x' || inputs.base_branch == '8.2.x' || inputs.base_branch == '9.0.x' }}
```
Comment (Progi1984) [inline .github/workflows/test-with-prebuilt-shop.yml:174]:
```suggestion
        if: ${{ inputs.base_branch == '8.0.x' || inputs.base_branch == '8.1.x' || inputs.base_branch == '8.2.x'}}
```

**Review by Progi1984 — COMMENTED** [review]
Ping @tleon for feedback

File: .github/workflows/actions/checkout-prestashop/action.yml (hunk 1 above)
Comment (jolelievre) [reply .github/workflows/actions/checkout-prestashop/action.yml:61]: yep its only needed for v8, so not develop nor 9.0

**Review by jolelievre — CHANGES_REQUESTED** [review]
Agree with the comments from @Progi1984

**Review by Progi1984 — APPROVED** [review]

**Pull request merged**